app-layout-addon/src/main/java/com/github/appreciated/app/layout/notification/DefaultNotification.java

```diff
@@ -12,14 +12,13 @@
 /**
  * Data structure that contains the information for a {@link Notification}
  */
-
 public class DefaultNotification implements Notification {
-    String title;
-    String description;
-    Resource image;
+    private String title;
+    private String description;
+    private Resource image;
     private Priority priority;
-    private boolean dismissible;
```

Comment:
Is "isIgnoreable" a better wording for that?

Reply:
You are a champ 👍.
> Is "isIgnoreable" a better wording for that?
I think "sticky" is in that case a more common word.